### ISSUE TITLE
Better seek bar for longer songs/live shows.

### ISSUE THREAD

**Issue by OutshineIssue** [25huizengek1/ViTune#337]
## Feature request:

The seek bar works fine for the average 4 minute song because if you want to listen to the last bit again, you drag the thumb a little to the left and that's it, a 10 seconds rewind, cool. The problem comes when you enjoy live shows like I do, where the recordings are 1 to 2 hours long, and the same drag action could mean an 8 minutes rewind. It's almost impossible to not only go back a few seconds, but to stay on the same song when barely dragging the thumb.

The simplest approach might be a customizable rewind button on the player. Another solution could be a precise seek bar —like the YouTube app does on longer videos— where, when dragging and holding, a zoomed sleek bar would let us move by the second. The last one seems like a good amount of work, so a rewind button would be just fine, I was just giving ideas.

**Comment by Monoxide8969** [comment]
This seems like a good feature to go with #260.

Unfortunately, as it's not planned in the near future, I think it would be best for you to listen to longer videos using newpipe

**Comment by OutshineIssue** [comment]
> This seems like a good feature to go with #260.

I can see the connection, but not really related.
 
> Unfortunately, as it's not planned in the near future, I think it would be best for you to listen to longer videos using newpipe

I guess you are talking about how _podcasts_ are not planned rn, but again, not related at all. This is merely a new button, there's even no need to rethink the UI for the addition of the new button, devs could just use the already present next/previous song and activate the rewind on long press.

The podcast thing is a whole different story.

**Comment by 25huizengek1** [comment]
@OutshineIssue Great idea, I'll come up with something.

**Comment by OutshineIssue** [comment]
@25huizengek1 Thanks, and also please don't forget about the [media controls](https://developer.android.com/media/implement/surfaces/mobile), it'll be really useful to rewind/fast forward from there too.

**Comment by OutshineIssue** [comment]
I noticed that the priority has been removed. What does that mean for the future of this issue? I would really love to see this feature implemented, as I mostly listen to live shows, and navigating through 2-hour videos can be quite challenging.

**Comment by Monoxide8969** [comment]
I don't think this is the right app for your usecase. Wouldn't you be more comfortable using newpipe (or any of the sponsorblock forks) or antennapod?

Not trying to be antagonistic, I just genuinely think vimusic has not been developed with this usecase in mind

**Comment by OutshineIssue** [comment]
Newpipe is a much larger project with a different scope and isn't really focused on music player UI/UX. AntennaPod doesn't have YouTube/YTMusic capabilities afaik? In reality, there isn't a dedicated music app designed for this specific use case to rely on.

> Not trying to be antagonistic, I just genuinely think vimusic has not been developed with this usecase in mind

@Monoxide8969 Exactly, but this isn't ViMusic; it's ViTune. Many forks of ViMusic and similar apps are too similar in features to the main repo, and they fail to establish a distinct identity, making their existence somewhat redundant.

This lack of differentiation is a missed opportunity. A fork should identify unmet user needs and improve upon them, capturing users that the main repo doesn't; otherwise, it just becomes a carbon copy with overlapped target users.
